tests/Feature/RequestMacroTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use Carbon\Carbon;
+use Surgiie\Transformer\Contracts\Transformable;
+use Surgiie\Transformer\DataTransformer;
+use Surgiie\Transformer\Exceptions\NotCallableException;
+use Surgiie\Transformer\Transformer;
+use function PHPUnit\Framework\greaterThan;
+
+beforeEach(function () {
+    Transformer::unguard();
+    // This is kind of a dirty way to add the macro
+    // without including orchestra/testbench
+    (new \Surgiie\Transformer\TransformerServiceProvider(''))
```

Comment:
Im okay with this for now since we're still able to test the functionality of the package without `orchestra/testbench`. Will leave it as a future consideration to pull in  `orchestra/testbench` if the package grows to the point where functionality cannot be tested without it.